packages/ibm-products/src/components/ExportModal/ExportModal.tsx

```diff
@@ -309,8 +309,12 @@ export let ExportModal = forwardRef(
           <div className={`${blockClass}__messaging`}>
```

Comment:
@anamikaanu96 ,
Would it be better if we give `aria-live="assertive"` to this div instead of each `P` tag items?